core/src/test/java/org/apache/calcite/test/RexProgramTest.java

```diff
@@ -2170,6 +2170,22 @@ private RexNode simplify(RexNode e) {
         is(false));
   }
 
+  @Test public void testIsAlwaysTrueAndFalse() {
+    final RelDataType booleanNullableType =
+        typeFactory.createTypeWithNullability(
+            typeFactory.createSqlType(SqlTypeName.BOOLEAN), true);
+    RexNode node = rexBuilder.makeInputRef(booleanNullableType, 0);
+
+    RexNode exp1 = rexBuilder.makeCall(SqlStdOperatorTable.IS_FALSE, isNotNull(isNull(node)));
```

Comment:
IS_FALSE should be refactored to a method like `isNull` for consistency and test brevity